latest_cov_build.py

```diff
@@ -89,3 +90,9 @@ def get_github_release_url(repo_slug):
         with tarfile.open(filename, mode) as tar:
             tar.extractall(path='tools')
     os.remove(filename)
+
+# Download Firefox coverage report
+codecoverage.download_coverage_artifacts(taskId, None)
+codecoverage.generate_info('tools/grcov')
```

Comment:
We'll need to modify this function in firefox-code-coverage to add an option to generate the report in another format.
By default, it's generating the report in the "INFO" format, but we want it in the json ("coveralls") format. This way we can use the diffing script you wrote for diffing the report from the crawler with this report from the tests.

Reply:
Ok, clear. I thought about this. Will try to modify this function.

Reply:
I've filed: https://github.com/marco-c/firefox-code-coverage/issues/30.